### PR TITLE
New servertype for pServer

### PR DESCRIPTION
**Pull request by sctnightcore**
```
2017-02-01aRagexeRE RagexeRE_2017_02_01a
2017-02-08bRagexeRE RagexeRE_2017_02_08b

```

### CODE REVIEW

File: src/Network/Send/kRO/RagexeRE_2017_02_08b.pm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		'0437' => ['character_move', 'a3', [qw(coordString)]],
+		'0892' => ['friend_request', 'a*', [qw(username)]],# len 26
+		'092D' => ['homunculus_command', 'v C', [qw(commandType, commandID)]],
+		'0921' => ['item_drop', 'a2 v', [qw(ID amount)]],
+		'0367' => ['item_list_window_selected', 'v V V a*', [qw(len type act itemInfo)]],
+		'0923' => ['item_take', 'a4', [qw(ID)]],
+		'088C' => ['map_login', 'a4 a4 a4 V C', [qw(accountID charID sessionID tick sex)]],
+		'085C' => ['party_join_request_by_name', 'Z24', [qw(partyName)]],
+		'083C' => ['skill_use', 'v2 a4', [qw(lv skillID targetID)]],
+		'0438' => ['skill_use_location', 'v4', [qw(lv skillID x y)]],
```
Comment (sctnightcore) [inline src/Network/Send/kRO/RagexeRE_2017_02_08b.pm]: need fix

File: src/Network/Send/kRO/RagexeRE_2017_02_01a.pm [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+		'0884' => ['character_move', 'a3', [qw(coordString)]],
+		'0919' => ['friend_request', 'a*', [qw(username)]],# len 26
+		'0938' => ['homunculus_command', 'v C', [qw(commandType, commandID)]],
+		'088B' => ['item_drop', 'a2 v', [qw(ID amount)]],
+		'0969' => ['item_list_window_selected', 'v V V a*', [qw(len type act itemInfo)]],
+		'0875' => ['item_take', 'a4', [qw(ID)]],
+		'094C' => ['map_login', 'a4 a4 a4 V C', [qw(accountID charID sessionID tick sex)]],
+		'0920' => ['party_join_request_by_name', 'Z24', [qw(partyName)]],
+		'083C' => ['skill_use', 'v2 a4', [qw(lv skillID targetID)]],
+		'0885' => ['skill_use_location', 'v4', [qw(lv skillID x y)]],
```
Comment (sctnightcore) [inline src/Network/Send/kRO/RagexeRE_2017_02_01a.pm]: need fix

File: src/Network/Send/kRO/RagexeRE_2017_02_08b.pm (hunk 1 above)
Comment (sctnightcore) [reply src/Network/Send/kRO/RagexeRE_2017_02_08b.pm]: fixed

File: src/Network/Send/kRO/RagexeRE_2017_02_01a.pm (hunk 2 above)
Comment (sctnightcore) [reply src/Network/Send/kRO/RagexeRE_2017_02_01a.pm]: fixed

File: src/Network/Send/kRO/RagexeRE_2015_09_16.pm [hunk 3]
```diff
@@ -37,7 +37,7 @@ sub new {
 		'0969' => ['map_login', 'a4 a4 a4 V C', [qw(accountID charID sessionID tick sex)]],
 		'0924' => ['party_join_request_by_name', 'Z24', [qw(partyName)]],
 		'093E' => ['skill_use', 'v2 a4', [qw(lv skillID targetID)]],
-		'0873' => ['skill_use_location', 'v4', [qw(lv skillID x y)]],
+		'022D' => ['skill_use_location', 'v4', [qw(lv skillID x y)]],
```
Comment (sctnightcore) [inline src/Network/Send/kRO/RagexeRE_2015_09_16.pm]: @moneyisgod  
Fixed

**Comment by alisonrag** [comment]
in my point of view is better add to sakexe_0 and keep only handler in ragexeRE

**Pull request merged**